src/providers/historical-telemetry-provider.js

```diff
@@ -60,6 +60,11 @@ export default class YamcsHistoricalTelemetryProvider {
     }
 
     getSampleHistory(id, start, end, size=300) {
+        // cap size at 1000, temporarily to prevent errors
+        if (size > 1000) {
```

Comment:
This is the right change. It looks like we also need to make the change in `getHistory`

Reply:
Oh dang... I'll get that.